### ISSUE TITLE
Handle asana sections

### ISSUE THREAD

**Issue by marcboscher** [marcboscher/crosscheck#33]
sections end with a colon
ignore them in sync?

**Comment by KyleChamberlin** [comment]
I find that using the colons as labels or milestones is useful. I have always sync'd them to milestones, however others might prefer labels. 

just a thought

**Comment by marcboscher** [comment]
yeah or people may want to add section in asana but not have them reflected in github

**Comment by KyleChamberlin** [comment]
true, I haven't had that pop up, but i can see use cases where that might be the case.
